README.md

```diff
@@ -32,7 +32,7 @@ The review documentation is for internal management discussion purposes only and
 | [Zenon Network](https://zenon.network/)| 05/05/2023     | [5204c0d](https://github.com/HyperCore-Team/evm-bridge-contracts/tree/5204c0df4e0a2a1bcaa69e5fa22c9131c09e76e9)     | [Report](/Zenon%20Network/zenon-05-2023.pdf)  |
 | [Gnosis Chain](https://www.gnosis.io/)| 25/05/2023     | [13e1555](https://github.com/gnosischain/deposit-contract/tree/13e155500b626612844e3d0fccc11b02b11ea785)     | [Report](/Gnosis%20Chain/gnosis-05-2023.pdf)  |
 | [EPNS](https://epns.io/)| 30/05/2023     | [993130c](https://github.com/ethereum-push-notification-service/push-smart-contracts/tree/993130c48211a83fd2b19fb082f2e25f91fa04cc)     | [Report](/EPNS/epns-protocol-05-2023.pdf)  |
-| [Hyperdrive](https://hyperdrive.delv.tech/)| 30/06/2023     | [1bcf5fe](https://github.com/delvtech/hyperdrive/commit/1bcf5fe45b9d3dd02741302dd639104338e79c21)     | [Report](/Hyperdrive/hyperdrive-06-2023.pdf)  |
+| [DELV](https://hyperdrive.delv.tech/)| 30/06/2023     | [1bcf5fe](https://github.com/delvtech/hyperdrive/commit/1bcf5fe45b9d3dd02741302dd639104338e79c21)     | [Report](/DELV/delv-06-2023.pdf)  |
```

Comment:
The github commit link is 404

Reply:
yeah, as we don't have control over their repos can't make sure sure relevant commits won't get lost somewhere.

Reply:
The link to their project page (hyperdrive.delv.tech) is 404 for me. Is the link correct?
(The Hyperdrive section on their website says "Coming Soon" so maybe it's not active yet)